gh-pages/content/index.md

```diff
@@ -17,6 +17,9 @@ languages will be added in the future!
 
     [runtime architecture]: overview/runtime-architecture.md
 
+!!! info
+    `jsii` [stands for](https://github.com/aws/jsii/issues/3490) **J**ava**S**cript **I**nteroperability **I**nterface
```

Comment:
It does not. "jsii" is not an acronym, same as "npm" is not an acronym (https://en.wikipedia.org/wiki/Npm).

MAYBE it _used_ to mean this, but this would be limiting future potential and anchors on current implementation details more than strictly necessary.

Reply:
Ok, i think the project should have a right to reject an acronym but that is a poor example as npm's acronym expansion is extremely obvious